test: Update allocated memory size value for K8s test

GabyCT opened the pull request:
Revert the memory value to the original value (https://github.com/kata-containers/tests/pull/534).
The main purpose of this is to fixed the bug (https://github.com/kata-containers/tests/issues/903)
where we could not allocated a memory size more than 100M.

Fixes #953

Signed-off-by: Gabriela Cervantes <gabriela.cervantes.tellez@intel.com>

GabyCT commented:
/test

grahamwhaley commented:
Hi @GabyCT I know you were investigating if/where we had a regression on this - looks like you found something? can you add details into probably both the Issue and the Commit so we can contemplate.
thx

GabyCT commented:
@grahamwhaley , yes I have been looking this, I tested around 12 commits (starting from Agust 30, which was the day this test start failing) for the runtime (using kubernetes version 1.10.5), however I could not find the regression. Then, I moved to the agent, where I tested also around 12 commits (build the agent using osbuilder), however, there I updated the kubernetes version to 1.12) and again I could not find the regression. Then, I checked with the latest agent, runtime and kubernetes version and I noticed that the test was working. I do not know if it was a combination of kubernetes version with maybe something else.

jodh-intel commented:
Hi @GabyCT - I'm not quite clear - Please could you summarise what is causing us to need to allocate 50M more in this test?

GabyCT commented:
@jodh-intel , I could not find the commit which introduce the regression (handle 100M), I just notice that with the latest changes we have 150M of allocated memory (size that we were using before)

ydjainopensource commented:
@GabyCT I don't understand. do you wish to say that we introduced an allocation somewhere which increased our memory footprint by 50MB and that s the reason why we need to change it here or are you saying that the newer version of k8ns needs this increase?

GabyCT commented:
@ydjainopensource , I am updating the value for the allocation of the memory before like we had before we had the regression. I noticed that with the latest agent, runtime and kubernetes version we can come back to the original amount of memory that we had before.

GabyCT commented:
/test

jodh-intel commented:
Hi @GabyCT - I'm still not totally clear what's going on here. It would be **really** helpful if you put details into the issue and the commit itself explaining this change as making seemingly arbitrary changes like this with no explanation is confusing to those (like me) who don't know the history of this file.

From what I can find out quickly, when you originally wrote this test, the original values was 150M:

- https://github.com/kata-containers/tests/pull/534

But then something happened and the value was modified (please provide references to what happened including issue(s)/PR(s)).

This PR is therefore reverting the value back to the original one I think?

GabyCT commented:
/test

GabyCT commented:
@jodh-intel , yes this PR is reverting the original value that we have from this test. The failures that we start seing was addressed in this issue https://github.com/kata-containers/tests/issues/680. As I mention above, we still do not know which PR introduced the regression as the runtime and agent were verified and we could not find anything.

jodh-intel commented:
I'm not very comfortable with the idea of making a change if we don't understand exactly why we're doing it.

At the very least though, please can you update the commit message to provide further details - it should explain:

- That the memory value is being reverted back to its original value of `150M` as "somehow", the bug(?) that was stopping 150M from being allocated has now been fixed / gone away.
- Provide a reference to the issue/PR where it was changed to `100M` (#805).

GabyCT commented:
@jodh-intel changes applied

jodh-intel commented:
CI failing due to the OSX brew bash issue here too (/cc @raravena80).

Thanks @GabyCT. Let's keep an eagle eye on this if we ever need to change it again.

lgtm

GabyCT commented:
/test

GabyCT commented:
/test

GabyCT commented:
/test

GabyCT commented:
/test

jodh-intel commented:
We need another ack over here @kata-containers/tests.
